tests/test_coordinates.py

```diff
@@ -194,6 +194,24 @@ def test_len(self):
         """len of Coords should return number of 3-vectors"""
         self.assertEqual(len(self.cvals), 2)
 
+    def test_SPH_output_noconvert(self):
+        """SPH is a synonym of GEO in spherical"""
+        self.cvals.ticks = Ticktock(['2002-02-02T12:00:00', '2002-02-02T12:00:00'], 'ISO')
+        expected = spc.Coords([[2, 45, 90], [3.0, 31, 31]], 'GEO', 'sph', use_irbem=False)
+        got = expected.convert('SPH', 'sph')
+        np.testing.assert_allclose(got.data, expected.data)
+        np.testing.assert_equal(got.dtype, 'GEO')
+
+    def test_SPH_output_convert(self):
+        """SPH is a synonym of GEO in spherical"""
+        self.cvals.ticks = Ticktock(['2002-02-02T12:00:00', '2002-02-02T12:00:00'], 'ISO')
+        expected = spc.Coords([[2, 45, 90], [3.0, 31, 31]], 'GEO', 'sph', use_irbem=False)
+        expected.ticks = self.cvals.ticks
```

Comment:
Similar to above...why not just assign those ticks directly to expected instead of going via cvals?